daemon/src/projection.rs

```diff
@@ -151,7 +151,7 @@ pub struct Cfd {
     pub contract_symbol: ContractSymbol,
     pub position: Position,
     #[serde(with = "round_to_two_dp")]
-    pub liquidation_price: Price,
+    pub liquidation_price: Decimal,
```

Comment:
I can add a dedicated type, but it didn't seem very useful since it currently won't be used to uphold any invariants or go through any layers.

Reply:
I started doing it and having `projection::Price` and `model::Price` is not great. We can't easily get rid of `model::Price` in, for example, `projection::Cfd` because it's used in calculations as we `apply` events.

I could just use the `projection::Price` for liquidation prices, but I'm not sure that's worth it. Feel free to give it a go yourself if you know how you'd like to improve the situation here.